packages/render-fidelity-tools/test/config.json

```diff
@@ -157,25 +137,206 @@
       }
     },
     {
-      "name": "khronos-Box",
-      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/Box/glTF-Binary/Box.glb",
+      "name": "khronos-BoxAnimated",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/BoxAnimated/glTF-Binary/BoxAnimated.glb",
       "orbit": {
         "theta": 30,
         "phi": 60,
         "radius": 2.38
+      }
+    },
+    {
+      "name": "khronos-CesiumMan",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/CesiumMan/glTF/CesiumMan.gltf",
+      "target": {
+        "y": 0.75
+      },
+      "orbit": {
+        "radius": 2
+      }
+    },
+    {
+      "name": "khronos-Cube",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/Cube/glTF/Cube.gltf",
+      "orbit": {
+        "theta": 30,
+        "phi": 60,
+        "radius": 5
+      }
+    },
+    {
+      "name": "khronos-Duck",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/Duck/glTF/Duck.gltf",
+      "target": {
+        "y": 0.86
+      },
+      "orbit": {
+        "radius": 2.7
+      }
+    },
+    {
+      "name": "khronos-InterpolationTest",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/InterpolationTest/glTF-Binary/InterpolationTest.glb",
+      "target": {
+        "y": 2.9
+      },
+      "orbit": {
+        "radius": 18.4
+      }
+    },
+    {
+      "name": "khronos-Lantern",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/Lantern/glTF/Lantern.gltf",
+      "dimensions": {
+        "height": 1024
+      },
+      "target": {
+        "x": 4,
+        "y": 13
+      },
+      "orbit": {
+        "radius": 39
+      }
+    },
+    {
+      "name": "khronos-MetalRoughSpheres",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/MetalRoughSpheres/glTF/MetalRoughSpheres.gltf",
+      "orbit": {
+        "radius": 12
+      },
+      "renderSkybox": true
+    },
+    {
+      "name": "khronos-MetalRoughSpheres-HDR",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/MetalRoughSpheres/glTF/MetalRoughSpheres.gltf",
+      "lighting": "../../../shared-assets/environments/spruit_sunrise_1k_HDR.hdr",
+      "orbit": {
+        "radius": 12
+      },
+      "renderSkybox": true
+    },
+    {
+      "name": "khronos-MetalRoughSpheres-LDR",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/MetalRoughSpheres/glTF/MetalRoughSpheres.gltf",
+      "lighting": "../../../shared-assets/environments/spruit_sunrise_1k_LDR.jpg",
+      "orbit": {
+        "radius": 12
+      },
+      "exclude": [
+        "dspbr-pt",
```

Comment:
Let's organize these so that all the ones with dspbr-pt are above all those without. We should also think about excluding any less-interesting scenarios from dspbr-pt so that we only pay for the rendering where the results are interesting.